### PR TITLE
zh-translation: .\docs\root\configuration\listeners\network_filters\t…

### PR DESCRIPTION
**Pull request by vgbhfive**
…cp_proxy_filter.rst

For an explanation of how to fill out the fields, please see the relevant section
in [PULL_REQUESTS.md](https://github.com/envoyproxy/envoy/blob/master/PULL_REQUESTS.md)

Commit Message:
Additional Description:
Risk Level:
Testing:
Docs Changes: .\docs\root\configuration\listeners\network_filters\tcp_proxy_filter.rst
Release Notes:
[Optional Runtime guard:]
[Optional Fixes #Issue] #242 
[Optional Deprecated:]

### CODE REVIEW

File: docs/root/configuration/listeners/network_filters/tcp_proxy_filter.rst [hunk 1]
```diff
@@ -1,64 +1,54 @@
...
 
-Dynamic cluster selection
+动态集群选择
 -------------------------
 
-The upstream cluster used by the TCP proxy filter can be dynamically set by
-other network filters on a per-connection basis by setting a per-connection
-state object under the key `envoy.tcp_proxy.cluster`. See the
-implementation for the details.
+TCP 代理过滤器使用的上游集群可以由其他网络过滤器根据每个连接动态设置，方法是在键 `envoy.tcp_proxy.cluster` 下设置每个连接的状态对象。见实现详情。
```
Comment (helight) [inline docs/root/configuration/listeners/network_filters/tcp_proxy_filter.rst]:
```suggestion
TCP 代理过滤器使用的上游集群可以由其它网络过滤器根据每个连接动态设置，方法是在 `envoy.tcp_proxy.cluster` 字段下设置每个连接的状态对象。见实现详情。
```

File: docs/root/configuration/listeners/network_filters/tcp_proxy_filter.rst [hunk 2]
```diff
@@ -1,64 +1,54 @@
...
 
 .. _config_network_filters_tcp_proxy_subset_lb:
 
-Routing to a subset of hosts
+路由到主机子集
 ----------------------------
 
-TCP proxy can be configured to route to a subset of hosts within an upstream cluster.
+TCP 代理可以被配置路由到上游集群内部的主机子集。
```
Comment (helight) [inline docs/root/configuration/listeners/network_filters/tcp_proxy_filter.rst]:
```suggestion
可以配置 TCP 代理，让它路由到上游集群内部的主机子集。
```

File: docs/root/configuration/listeners/network_filters/tcp_proxy_filter.rst [hunk 3]
```diff
@@ -1,64 +1,54 @@
...
 
-To define metadata that a suitable upstream host must match, use one of the following fields:
+为了定义合适的上游主机必须匹配的元数据，使用以下字段：
```
Comment (helight) [inline docs/root/configuration/listeners/network_filters/tcp_proxy_filter.rst]:
```suggestion
为了定义适合的上游主机必须匹配的元数据，请使用以下字段之一：
```

File: docs/root/configuration/listeners/network_filters/tcp_proxy_filter.rst [hunk 4]
```diff
@@ -1,64 +1,54 @@
...
 .. _config_network_filters_tcp_proxy_stats:
 
-Statistics
+统计信息
 ----------
 
-The TCP proxy filter emits both its own downstream statistics as well as many of the :ref:`cluster
-upstream statistics <config_cluster_manager_cluster_stats>` where applicable. The downstream
-statistics are rooted at *tcp.<stat_prefix>.* with the following statistics:
+TCP 代理过滤器同时发出它自己的下游统计信息和许多如果可用的 :ref:`上游集群统计信息 <config_cluster_manager_cluster_stats>` 。下游统计信息以 *tcp.<stat_prefix>.* 为根，如下所示的统计信息：
```
Comment (scilla0531) [inline docs/root/configuration/listeners/network_filters/tcp_proxy_filter.rst]: TCP 代理过滤器既会发出自己的下游统计信息，也会在适用的情况下发出许多 :ref:`集群上游统计信息 <config_cluster_manager_cluster_stats>` 。下游统计信息以 *tcp.<stat_prefix>.* 为根，统计信息如下：

**Review by helight — APPROVED** [review]
LGTM

**Review by scilla0531 — APPROVED** [review]

**Pull request merged**